airbyte-cdk/python/unit_tests/sources/declarative/auth/test_oauth.py

```diff
@@ -149,6 +149,7 @@ def test_refresh_access_token_expire_format(self, mocker, expires_in_response, t
             scopes=["scope1", "scope2"],
             token_expiry_date="{{ config['token_expiry_date'] }}",
             token_expiry_date_format=token_expiry_date_format,
+            token_expiry_is_time_of_expiration=True,
```

Comment:
there should be a test for the other case (token_expiry_is_time_of_expiration=False)

Reply:
Looks like the [`airbyte-cdk/python/unit_tests/sources/declarative/auth/test_oauth.py test_set_token_expiry_date_no_format`](https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/unit_tests/sources/declarative/auth/test_oauth.py#L181) covers this case